src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -122,6 +124,8 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
     this.vis.uiState.on('change', this.uiStateChangeHandler);
     this.vis.uiState.on('reload', this.reload);
 
+    this.inspectorAdapters = this.vis.type.inspectorAdapters;
```

Comment:
remove this line, you set this.inspectorAdapters on line 142

Reply:
thank you, forgot to remove